### DISCUSSION TITLE
How to change/override the font of CCP?

### DISCUSSION THREAD

**Discussion by shayanbagchi** [hbb20/CountryCodePickerProject#558] [Q&A]
Hello everyone,

I'm trying to change/override the font family of the CCP, but `android:fontFamily="@font/spartan_regular"` is not working.
If someone knows how to do it, please help me with this.

Thanks

**Comment by hbb20** [comment]
Did you try https://github.com/hbb20/CountryCodePickerProject/wiki/Custom-TypeFace-(FontFamily) ?
